metadata.yaml

```diff
@@ -14,3 +14,6 @@ releaseSeries:
   - major: 0
     minor: 4
     contract: v1beta1
+  - major: 0
+    minor: 5
+    contract: v1beta2 
```

Comment:
this should be `v1beta1` the version of a capi api we have contract with

Reply:
Got it, Addressed 👍🏼